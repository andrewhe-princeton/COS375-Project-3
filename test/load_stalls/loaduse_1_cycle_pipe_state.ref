Cycle:        0	|| lw $t0, 0($t0)          | nop                     | nop                     | nop                     | nop                     |
Cycle:        1	|| lw $t0, 0($t0)          | nop                     | nop                     | nop                     | nop                     |
Cycle:        2	|| lw $t0, 0($t0)          | nop                     | nop                     | nop                     | nop                     |
Cycle:        3	|| lw $t0, 0($t0)          | nop                     | nop                     | nop                     | nop                     |
Cycle:        4	|| lw $t0, 0($t0)          | nop                     | nop                     | nop                     | nop                     |
Cycle:        5	|| lw $t0, 0($t0)          | nop                     | nop                     | nop                     | nop                     |
Cycle:        6	|| add $t0, $t0, $t0       | lw $t0, 0($t0)          | nop                     | nop                     | nop                     |
Cycle:        7	|| HALT                    | add $t0, $t0, $t0       | lw $t0, 0($t0)          | nop                     | nop                     |
Cycle:        8	|| HALT                    | add $t0, $t0, $t0       | nop                     | lw $t0, 0($t0)          | nop                     |
Cycle:        9	|| HALT                    | add $t0, $t0, $t0       | nop                     | lw $t0, 0($t0)          | nop                     |
Cycle:       10	|| HALT                    | add $t0, $t0, $t0       | nop                     | lw $t0, 0($t0)          | nop                     |
Cycle:       11	|| HALT                    | add $t0, $t0, $t0       | nop                     | lw $t0, 0($t0)          | nop                     |
Cycle:       12	|| HALT                    | add $t0, $t0, $t0       | nop                     | lw $t0, 0($t0)          | nop                     |
Cycle:       13	|| HALT                    | add $t0, $t0, $t0       | nop                     | lw $t0, 0($t0)          | nop                     |
Cycle:       14	|| HALT                    | add $t0, $t0, $t0       | nop                     | lw $t0, 0($t0)          | nop                     |
Cycle:       15	|| HALT                    | add $t0, $t0, $t0       | nop                     | lw $t0, 0($t0)          | nop                     |
Cycle:       16	|| HALT                    | add $t0, $t0, $t0       | nop                     | lw $t0, 0($t0)          | nop                     |
Cycle:       17	|| nop                     | HALT                    | add $t0, $t0, $t0       | nop                     | lw $t0, 0($t0)          |
Cycle:       18	|| nop                     | nop                     | HALT                    | add $t0, $t0, $t0       | nop                     |
Cycle:       19	|| nop                     | nop                     | nop                     | HALT                    | add $t0, $t0, $t0       |
Cycle:       20	|| nop                     | nop                     | nop                     | nop                     | HALT                    |
