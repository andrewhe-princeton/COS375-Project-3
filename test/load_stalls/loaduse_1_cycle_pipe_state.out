Cycle:        0	|| lw $t0, 0($t0)          | nop                     | nop                     | nop                     | nop                     |
Cycle:        1	|| lw $t0, 0($t0)          | nop                     | nop                     | nop                     | nop                     |
Cycle:        2	|| lw $t0, 0($t0)          | nop                     | nop                     | nop                     | nop                     |
Cycle:        3	|| lw $t0, 0($t0)          | nop                     | nop                     | nop                     | nop                     |
Cycle:        4	|| lw $t0, 0($t0)          | nop                     | nop                     | nop                     | nop                     |
Cycle:        5	|| lw $t0, 0($t0)          | nop                     | nop                     | nop                     | nop                     |
Cycle:        6	|| add $t0, $t0, $t0       | lw $t0, 0($t0)          | nop                     | nop                     | nop                     |
Cycle:        7	|| HALT                    | add $t0, $t0, $t0       | lw $t0, 0($t0)          | nop                     | nop                     |
Cycle:        8	|| HALT                    | add $t0, $t0, $t0       | nop                     | lw $t0, 0($t0)          | nop                     |
Cycle:        9	|| nop                     | HALT                    | add $t0, $t0, $t0       | nop                     | lw $t0, 0($t0)          |
Cycle:       10	|| nop                     | nop                     | HALT                    | add $t0, $t0, $t0       | nop                     |
Cycle:       11	|| nop                     | nop                     | nop                     | HALT                    | add $t0, $t0, $t0       |
Cycle:       12	|| nop                     | nop                     | nop                     | nop                     | HALT                    |
